### DISCUSSION TITLE
Gen I & II VC Support

### DISCUSSION THREAD

**Discussion by tdadvocate** [Universal-Team/pkmn-chest#73] [Ideas]
I hate to be "that guy" but I was just wondering if this would ever support the 3DS' VC versions of Gen I & II. Would be really nice to have an all in one tool for especially when the eShop Servers are gone for good and then eventually stuff like PokéBank and Transporter dying as well.

**Comment by lifehackerhansol** [comment]
DSi mode doesn't have access to CTRNAND so this will not be possible.

PKSM, which does, would need to implement it. There is an ongoing pull request to do so, which actually looks complete but nothing has been done about it yet: https://github.com/FlagBrew/PKSM-Core/pull/4 https://github.com/FlagBrew/PKSM/pull/1297
